quiz.js

```diff
@@ -0,0 +1,44 @@
+var grow= [];
+var branchNum= document.getElementById("branches");
+var leafNum= document.getElementById("leaf");
+var button = document.getElementById("button");
+
+function tree(branches, leaf) {
+	if (branchNum.value==="" || leafNum.value===""){
+		alert("Both fields must have a value");
+		} else{
+		for(j=0; j<branches; j++) {
+			grow.unshift(" ");
+		}
+		for(i=0; i<branches; i++) {
+			grow.push(leaf);
```

Comment:
As is, your tree isn't quite right.  If you check the instructions, your tree starts with a top that's 2 'leaves' wide, whereas the example tree is only one wide.